tests/test_model_functions.py

```diff
@@ -62,6 +62,24 @@ def test_graph_for_dae(self):
     #######################################################################
     # this section contains tests for each get_XXX function on Model
 
+    def test_get_derived_quantities(self, basic_model):
+        """ Tests Model.get_state_symbols() works correctly. """
+
+        derived_quantities = basic_model.get_derived_quantities()
+        assert str(derived_quantities) == '[Derivative(_env_ode$sv1, _environment$time)]'
```

Comment:
I thought this would be excluded as it's  a state?

Reply:
`_environment$time` is but the derivative isn't it is a deriv of course.